sample-app/src/components/InputDropdown.tsx

```diff
@@ -97,24 +121,43 @@ export default function InputDropdown({
       evt.preventDefault();
     }
 
+    const isFirstSectionFocused = focusedSectionIndex === 0;
```

Comment:
Adding sections support is adding a lot of complexity to this logic. If we support either an array of options or a sections object, then we could have separate functions where one handles the logic for simple options, and the other handles the logic for sections.